x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -275,8 +323,10 @@ export function dataFrameAnalyticsRoutes({ router, mlLicense }: RouteInitializat
   router.delete(
     {
       path: '/api/ml/data_frame/analytics/{analyticsId}',
+      // validate: false,
```

Comment:
please remove commented code

Reply:
Fixed in https://github.com/elastic/kibana/pull/66934/commits/d14fafab86d64ef3afce4bda0f3c415ea85e672e